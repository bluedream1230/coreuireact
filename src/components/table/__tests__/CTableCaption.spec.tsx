import * as React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CTableCaption } from '../../../index'

test('loads and displays CTableCaption component', async () => {
  const { container } = render(<CTableCaption/>)
  expect(container).toMatchSnapshot()
})

test('CTableCaption customize', async () => {
    const { container } = render(
        <CTableCaption>
            Test
        </CTableCaption>
    )
    expect(container).toMatchSnapshot()
    expect(container.firstChild).toHaveTextContent('Test')
})