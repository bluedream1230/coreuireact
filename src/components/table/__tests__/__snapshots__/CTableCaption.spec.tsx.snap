// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTableCaption customize 1`] = `
<div>
  <caption>
    Test
  </caption>
</div>
`;

exports[`loads and displays CTableCaption component 1`] = `
<div>
  <caption />
</div>
`;
